src/python/platforms/android/adb.py

```diff
@@ -424,6 +442,25 @@ def reset_device_connection():
 
   return True
 
+def get_gce_device_ip():
+  try:
+    return socket.gethostbyname('cuttlefish')
+  except socket.gaierror as _:
+    logs.log_warn('Failed to get cuttlefish ip address, use localhost instead')
+    return '127.0.0.1'
+
+
+def set_gce_device_serial():
+  device_serial = '%s:6520' % get_gce_device_ip()
```

Comment:
Can we put 6520 into a constant somewhere? 

And then use f-strings to build the resulting string.

Reply:
Done